src/main/templates/com/smartbear/postman/PluginConfig.java

```diff
@@ -28,4 +31,10 @@
     infoUrl = "${project.url}",
     minimumReadyApiVersion = "${ready-api-version}")
 public class PluginConfig extends PluginAdapter {
+
+    public PluginConfig() {
+        super();
+        ApplicationEnvironment.getSoapUICore().getFactoryRegistry().addFactory(ImportMethodFactory.class,
+                new PostmanImportMethodFactory());
```

Comment:
Is it a common practice to add plugin's factory to a registry from plugin's code? Shouldn't we implement a new annotation and move this logic to PluginLoader instead? I'm not sure when this constructor is called - is it possible that Postman section will be added in a random place of the dialog?

Reply:
> Is it a common practice to add plugin's factory to a registry from plugin's code?

I amn't sure about that. But please note that the method ``addFactory`` is designed to be used for adding factories. We have access to these classes from plug-ins so I believe it is OK.

> Shouldn't we implement a new annotation and move this logic to PluginLoader instead?

Not sure it is better. In general ``addFactory`` seems to be a standard way.

> is it possible that Postman section will be added in a random place of the dialog?

No, that isn't possible. The factories are gotten and used from the core when the dialog is open by some user action (i.e. much later after the core is initialized).

Reply:
> The factories are gotten and used from the core when the dialog is open by some user action (i.e. much later after the core is initialized).
I mean a different thing. Are you sure this addFactory() in PluginConfig() will be called after all "core" ImportMethodFactory implementations are added (so that Postman won't end up between WADL and Swagger options in the registry)?

Reply:
Ok, thanks for clarification.